docs/guides/postgres/branch/customization/index.md

```
@@ -0,0 +1,536 @@
+dev-branch-refresh   Refreshing   Local   dev-postgres-refresh   3m40s       10m
+dev-branch-refresh   Refreshing   Local   dev-postgres-refresh   5m          11m
+dev-branch-refresh   Refreshing   Local   dev-postgres-refresh   0s          11m
+dev-branch-refresh   Ready        Local   dev-postgres-refresh   0s          11m
+```
+
+`FRESHNESS` climbs while the branch sits on its old copy and resets to `0s` the moment the new one is live.
+
+Behind that transition, Courier took a new source snapshot, halted the branch, dropped its old volume, cloned a new one, and brought the database back up. The status shows the bookkeeping:
+
+```bash
+$ kubectl get branch -n demo dev-branch-refresh -o yaml
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)